build.gradle

```diff
@@ -3,7 +3,7 @@ task wrapper(type: Wrapper) {
 }
 
 allprojects {
-    version = "2.0.6"
+    version = "2.0.7"
```

Comment:
Should we mirror the `exoplayer` versioning and consider this a `2.1.0` release rather than `2.0.7`?